src/solc_0.8/Game/GameMinter.sol

```diff
@@ -21,54 +22,49 @@ contract GameMinter is WithMetaTransaction, IGameMinter {
 
     constructor(
         GameToken gameTokenContract,
-        address metaTransactionContract,
+        address trustedForwarder,
         uint256 gameMintingFee,
         uint256 gameUpdateFee,
         address feeBeneficiary,
         IERC20 sand
-    ) {
+    ) ERC2771Context(trustedForwarder) {
```

Comment:
The ERC2771 context use an immutable variable for the trusted forwarder which is great for optimization and could still be updated if our contract is deployed as upgradeable proxy

But this also means : we cannot have multiple forwarder support at once
we cannot make the contract immutable unless we also agree to never change the forwarder

Just mentionning these here, but I guess we are planning to go with the "upgradeable proxy" route anyway

Reply:
Indeed. Yeah, GameToken is not currently set up to be upgradable as-is... I imagine we want to have that capability, but not sure if we ever got a firm answer on that?

Reply:
the more I think about it, the more I think Upgradeable contract will be inevitable for the foresable future as we do not yet how we will be able to handle bridging nicely across different L2 solution.

So yes, let's make the deployment via proxy too

Reply:
Sounds good, and agreed that we'll probably need it.

Reply:
@wighawag I would submit a separate PR making the game token upgradable.